entrypoint.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -e
+while getopts "a:b:c:d:e:f:g:h:i:j:" o; do
+   case "${o}" in
+       a)
+         export scanType=${OPTARG}
+       ;;
+       b)
+         export format=${OPTARG}
+       ;;
+       c)
+         export template=${OPTARG}
+       ;;
+       d)
+         export exitCode=${OPTARG}
+       ;;
+       e)
+         export ignoreUnfixed=${OPTARG}
+       ;;
+       f)
+         export vulnType=${OPTARG}
+       ;;
+       g)
+         export severity=${OPTARG}
+       ;;
+       h)
+         export output=${OPTARG}
+       ;;
+       i)
+         export imageRef=${OPTARG}
+       ;;
+       j)
+         export scanRef=${OPTARG}
+       ;;
+  esac
+done
+
+scanType=$(echo $scanType | tr -d '\r')
+export artifactRef="${imageRef}"
+if [ "${scanType}" = "fs" ];then
+  artifactRef=$(echo $scanRef | tr -d '\r')
+fi
```

Comment:
What does this do?

Reply:
basically since we wanted to support image-ref, I am setting the variable artifactRef with it, but override it with scan-ref when type is fs, and that `tr -d '\r'` is because the options were parsing `\r` character causing 
```
if [ "${scanType}" = "fs" ]
```
to fail. I did many testing and found that we need to remove that char before checking condition.